website/docs/docs/cloud/about-cloud/dark-mode.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "dbt Cloud dark mode"
```

Comment:
My suggestion would be to make this page light/dark agnostic. 

Something like "Change your dbt Cloud Theme" or similar. @runleonarun and @mirnawong1 what are your thoughts here

Reply:
i think that's a good shout @matthewshaver